docs/source/markdown/podman-network.1.md

```diff
@@ -10,11 +10,11 @@ podman\-network - Manage Podman networks
 The network command manages networks for Podman.
 
 Podman supports two network backends [Netavark](https://github.com/containers/netavark)
-and [CNI](https://www.cni.dev/). Netavark is the default network backend and was added in Podman v4.0. CNI will be
-deprecated in the future in preference of Netavark. To configure
+and [CNI](https://www.cni.dev/). Netavark is the default network backend and was added in Podman v4.0. CNI is deprecated in the future in preference of
```

Comment:
This is one place it can't change.
```suggestion
and [CNI](https://www.cni.dev/). Netavark is the default network backend and was added in Podman v4.0. CNI will be deprecated in the future in preference of
```